src/index.html

```diff
@@ -10,8 +10,11 @@
 <body class="guest-js">
```

Comment:
Don't forget hour meta viewport info in head.

Reply:
What do you mean by this?